index.html

```diff
@@ -57,6 +57,7 @@ <h2 class="title text-center text-capitalize">Cookie Manager</h2>
                 <input type="text" class="form-control" id="value" required="required">
               </div>
             <button type = "button"  class="btn btnBelow btn-danger" id="clear_cookies">Clear All Cookies</button>
+            <button type="button" class="btn btnBelow" id="delete_domain_cookies">Delete Domain Cookies</button>
```

Comment:
Please, use any bootstrap button classes.

Reply:
Right. I've left it unstyled besides the font-declaration. What color were you after, red like the other one?

Reply:
As it's `delete` button, **red will look good**.

Reply:
fixed